packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -629,6 +631,15 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
         setCenter(center);
     }, [center]);
 
+    /**
+     * Sets app title.
+     */
```

Comment:
```suggestion
    /*
     * Sets document title based on useAppTitle and appConfig values.
     */
```

It's not a function definition, but a function call. So no JSDoc notation.

Mr.Elite Linter Expert, @Lucaci-Andrei, do you think such a eslint rule exists?

Reply:
I do believe we might have something 🧐

Reply:
Update: unfortunately I could not find a rule that would work only on multiline comments 🤔 . We would need to create a custom rule for it.